addons/l10n_hr_edi/models/l10n_hr_edi_addendum.py

```diff
@@ -0,0 +1,66 @@
+from odoo import models, fields
+
+
+class L10nHrEdiAddendum(models.Model):
+    _name = 'l10n_hr.edi_addendum'
```

Comment:
I think it would make more sense to call this model `l10n_hr_edi.document` just because we already have lots of `l10n_xx_edi.document` models (but maybe there was a good reason to call it `addendum` rather than `document`?)

```suggestion
class L10nHrEdiDocument(models.Model):
    _name = 'l10n_hr_edi.document'
```